nym-wallet/src/components/IdentityKey.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import { Stack, Typography } from '@mui/material';
+import { Stack, Typography, Tooltip } from '@mui/material';
 import { CopyToClipboard } from '@nymproject/react/clipboard/CopyToClipboard';
 import { splice } from 'src/utils';
 
-export const IdentityKey = ({ identityKey }: { identityKey: string }) => (
-  <Stack direction="row">
-    <Typography variant="body2" component="span" fontWeight={400} sx={{ mr: 1, color: 'text.primary' }}>
-      {splice(6, identityKey)}
-    </Typography>
-    <CopyToClipboard value={identityKey} sx={{ fontSize: 18 }} />
-  </Stack>
+export const IdentityKey = ({ identityKey, tooltipTitle }: { identityKey: string; tooltipTitle?: string }) => (
+  <Tooltip title={tooltipTitle || ''} placement="top" arrow>
+    <Stack direction="row" width="fit-content">
+      <Typography variant="body2" component="span" fontWeight={400} sx={{ mr: 1, color: 'text.primary' }}>
+        {splice(6, identityKey)}
+      </Typography>
+      <CopyToClipboard value={identityKey} sx={{ fontSize: 18 }} />
+    </Stack>
+  </Tooltip>
 );
```

Comment:
Is it possible to wrap <IdentityKey /> in a tooltip instead of passing in this extra prop and wrapping inside the component?

Reply:
@fmtabbara It doesn't work. I've been trying fist to wrap directly with the tooltip, and I didn't find the way to make it work.  But wrapping directly the component it does